apps/trackers/tests/test_common.py

```diff
@@ -448,10 +449,12 @@ def test_bugzilla_virtualizaiton(self, ps_component):
         )
 
     @pytest.mark.parametrize("flaw_count", [1, 2, 3])
-    def test_community(self, flaw_count):
+    def test_community(self, monkeypatch, flaw_count):
         """
         test community tracker description generation
         """
+        monkeypatch.setattr(commom, "BZ_URL", "https://example.com")
```

Comment:
maybe we should use an auto-use fixture instead? that way any test/code that depends on these vars will already be monkey patched implicitly